src/GitHub.Exports/Services/Services.cs

```diff
@@ -120,5 +121,28 @@ public static string GetFileNameFromActiveDocument()
             var fullName = Dte2?.ActiveDocument?.FullName;
             return Path.GetFileName(fullName);
         }
+
+        /// <summary>
+        /// There doesn't appear to be a command that directly opens a target repo.
+        /// Our workaround is to create, open and delete a solution in the repo directory.
+        /// This triggers an event that causes the target repo to open. ;)
+        /// </summary>
+        /// <param name="directory">The target repo directory. </param>
+        public static void OpenRepository(string directory)
```

Comment:
`VSServices` is a better place for this method, the `Services` class is (should be) restricted to helper properties that are needed outside of a MEF context, and it just needs to die in a :fire: